net/akq.c

```diff
@@ -73,7 +75,7 @@ static void akq_worker_int(struct akq *q)
 {
     struct akq_work_list work_list;
     TAILQ_INIT(&work_list);
-    while (1) {
```

Comment:
This is wrong - nothing will ever resume akq after `start_threads` is called. We never want to stop processing replication net traffic - that is where subsequent start-thread will be received!
akq shutdown is performed by `stop_event_net`.